kolibri/content/serializers.py

```diff
@@ -96,35 +46,23 @@ def __init__(self, *args, **kwargs):
         super(ContentNodeSerializer, self).__init__(*args, **kwargs)
 
         # enable dynamic fields specification!
-        if not isinstance(self.context['request'], WSGIRequest):
```

Comment:
why delete this check? will get this error `'WSGIRequest' object has no attribute 'query_params'` when render the explore page